clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -842,11 +844,13 @@ class GCEUtil {
           } else {
             throw jsonException
           }
-        } catch (GoogleOperationTimedOutException __) {
+        } catch (GoogleOperationTimedOutException timeout) {
+          t = timeout
           log.warn "Retry $action timed out again, trying again..."
         }
       }
-      throw new GoogleOperationException("Failed to delete $resource after #$tries")
+      log.warn("Failed to $action $resource after #$tries.")
+      throw t
```

Comment:
There's at least a log trace of the error codes. I could also log the messages, but that would get quite noisy.